source/androidx.datastore/datastore-core/Transforms/Metadata.xml

```diff
@@ -0,0 +1,15 @@
+﻿<metadata>
+    <attr
+        path="/api/package[@name='androidx.datastore.core']"
+        name="managedName"
+        >
+        AndroidX.DataStore.Core
```

Comment:
Can we use:
```xml
<AndroidNamespaceReplacement Include='datastore' Replacement='DataStore' />
``` 
instead of all of these metadata files?

Reply:
Metadata removed.  `AndroidNamespaceReplacement` item added